docs/streams/upgrade-guide.html

```diff
@@ -198,6 +198,16 @@ <h3><a id="streams_api_changes_360" href="#streams_api_changes_360">Streams API
     </code>
     </pre>
     </p>
+
+    <p>
+        KStreams-to-KTable joins now have an option for adding a grace period.
+        The grace period is enabled on the <code>Joined</code> object using with <code>withGracePeriod()</code> method.
+        This change was introduced in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-923%3A+Add+A+Grace+Period+to+Stream+Table+Join">KIP-923</a>.
+        To use the grace period option in the Stream-Table join the table must be versioned, see the
+        <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html#versioned-state-stores">Developer Guide</a>.
+        For more information, including how it can be enabled and further configured, see the <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html#kstream-ktable-join"><b>Kafka Streams Developer Guide</b></a>.
```

Comment:
```
the table must be versioned, see the
        <a href="/{{version}}/documentation/streams/developer-guide/dsl-api.html#versioned-state-stores">Developer Guide</a>.        
```

Use `versioned` as link and remove the `, see the Developer Guide` part.

(This change also avoid the usage of `Developer Guide` in a similar way in two consecutive sentences.